yarn-project/stdlib/src/logs/l1_payload/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './encrypted_log_payload.js';
-export * from './l1_note_payload.js';
+export * from './shared_secret_derivation.js';
```

Comment:
Can't we get rid of this entire directory?

Reply:
We cannot because there is still l1_event_payload that uses this stuff.

I was thinking about this and I think we should prioritize moving the event decryption to Noir as well once this PR is merged because I currently have it all loaded in my brain and it would allows us to finally purge a lot of this code.

It would require modifying the process_log function such that it stores the logs in capsules and then exposing some event getter on the relevant contracts so it would be quite a bit of work though.